lib/rmt/gpg.rb

```diff
@@ -0,0 +1,51 @@
+require 'English'
+require 'fileutils'
+require 'tmpdir'
+
+class RMT::Gpg
```

Comment:
I think it should be `RMT::GPG` everywhere

Reply:
OK, fair enough.

Reply:
Done.